File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
      int[] output = new int[nums.length];
      Arrays.fill(output,1); 
      int i = 0;
      while(i < nums.length){
        for(int j = 0; j < i; j++){
            int left_prod = 1;
            left_prod = left_prod * nums[j];
            output[i] = output[i]*left_prod;
        }
        for(int j = i+1; j < nums.length ; j++){
            int right_prod = 1;
            right_prod = right_prod * nums[j];
            output[i] = output[i]*right_prod;
        }
        i++;
      }
      return output;
    }
}  

File: Data Structures & Algorithms/duplicate-integer/submission-1.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        if len(nums) == 0 or len(nums) == 1:
            return False
        occ = {}
        for num in nums:
            if num in occ:
                occ[num] += 1
            else:
                occ[num] = 1
        for v in occ.values():
            if v > 1:
                return True
        return False